level-ttl.js

```diff
@@ -183,7 +208,7 @@ function batch (db, arr, options, callback) {
     on  = []
     off = []
     arr.forEach(function (entry) {
-      if (!entry || entry.key === null || entry.key === undefined)
+      if (!entry || entry.key === null || entry.key === undefined || entry.key.indexOf('\xffttl\xff') > -1)
```

Comment:
This is normal behaviour I guess since `batch` is called both internally to manage ttl keys (`put` and `del`) and externally. The problem is that if there are no options and `defaultTTL` is used - which is essentially a global state - the ttl-if-statement in `level-ttl#batch` will be triggered even for *internal* batch calls.

So there are two options to this:

1. Check for keys and ignore keys that has to do with ttl logic. Which right now is also a leaky abstraction, since we are assuming that `level-spaces` are using `\xff` as a separator.

2. Why is `level-updown` even calling this batch function in the first place? When doing internal batch calls, we should use the default `batch` method of `levelup` and not the `level-ttl` version. So I would be inclined to say that this is actually a bug. We shouldn't need to do this key check in the first place.